index.js

```diff
@@ -18,6 +42,16 @@ async function uuid(username){
     return response;
 }
 
+var special_ids = {
```

Comment:
There ID's should be for Mccapes staff roles

478670065483513860 //support staff
479048180202340362 //contributor
628211166321311744 //cape creative